roles/testnode/tasks/user.yml

```diff
@@ -3,6 +3,8 @@
   group:
     name: sudo
     state: present
+  tags:
+   - user
```

Comment:
The include in main.yml tags everything in this file as 'user' already.  These tags are unneeded.